Sources/PackageCollections/Storage/SQLitePackageCollectionsStorage.swift

```diff
@@ -39,29 +38,27 @@ final class SQLitePackageCollectionsStorage: PackageCollectionsStorage, Closable
     private let stateLock = Lock()
 
     private let cache = ThreadSafeKeyValueStore<Model.CollectionIdentifier, Model.Collection>()
-    private let cacheLock = Lock()
-
-    private let isShuttingdown = ThreadSafeBox<Bool>()
 
     // Lock helps prevent concurrency errors with transaction statements during e.g. `refreshCollections`,
     // since only one transaction is allowed per SQLite connection. We need transactions to speed up bulk updates.
     // TODO: we could potentially optimize this with db connection pool
     private let ftsLock = Lock()
     // FTS not supported on some platforms; the code falls back to "slow path" in that case
-    let useSearchIndices = ThreadSafeBox<Bool>()
+    internal let useSearchIndices = ThreadSafeBox<Bool>() // internal for testing
```

Comment:
This is not for internal testing only--in case FTS is not supported this is set to `false` and the code falls back to querying the db tables.

Reply:
sry I meant the variable is internal vs private